src/iris/api.py

```diff
@@ -4240,7 +4251,7 @@ def on_post(self, req, resp):
         try:
             app, response = self.handle_user_response('email', msg_id, source, cmd)
         except Exception:
-            logger.exception('Failed to handle email response: %s' % first_line)
+            logger.warning('Failed to handle email response: %s' % first_line)
```

Comment:
Why this change? Don't we want to log the exception?

Reply:
This was causing a ton of noise in the logs due to users responding with an incorrect message id. Since this can fail in several ways from user input and is not critical I think logging an exception is overkill.